src/styles/00-base/01-colors/_color-vars.scss

```diff
@@ -35,3 +35,5 @@ $nypl-blue-dark: #004b98;
 $nypl-purple-light: #daa9e3;
 $nypl-purple-regular: #8a1a9c;
 $nypl-purple-dark: #673165;
+
+$rn-gray: #d8d8d8
```

Comment:
Can you please remove this and use a pre-existing gray? (`$gray-medium` would work.) If this doesn't already exist in our color schema that's something that I would prefer the designers to think about holistically. Grays are particularly egregious in how they multiply.

I've added an [open issue](http://themetronome.co/components/ol-32-text-link-w-icon-list/?fresh=true) to the documentation site.